docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md

```diff
@@ -79,6 +79,7 @@ Create the following environment variables and set to the specified values:
 * `BESU_NODE_PERM_ENDPOINT` - required only if your node is not using the default JSON-RPC host and
   port (`http://127.0.0.1:8545`). Set to JSON-RPC host and port. When bootstrapping the network,
   Besu uses the specified node to deploy the contracts and is the first node in the network.
+* `REACT_APP_NETWORK_ID` ID of the network. The chainID from the genesis file.
```

Comment:
Is is the chain ID or the network ID that's required?  ie, if you've got a network where you've overridden the chain ID in the genesis file using the `--network-id` option, do you specify the network ID here?

Reply:
it is definitely chain ID. network ID is not used here

Reply:
it is definitely chain ID

Reply:
renamed the environment variable (pending PR on permissioning repo)